lib/tasks/upload_source_strings_for_translation.rake

```diff
@@ -0,0 +1,57 @@
+# This rake task takes in a model name as a string. It is case sensitive.
+# So, a model class named "Foo" will error if "foo" is passed in as the argument.
+# For models that are namespaced, e.g. "Foo::BarBaz",
+# the string argument would be "Foo_BarBaz".
+namespace :i18n_sonder do
+  desc "Upload the source strings for all objects of a given model type for translation"
+  task :upload_source_strings_for_translation, [:model_name] => :environment do |_task, args|
+
+    if args[:model_name].present?
+      klasses = [args[:model_name].constantize]
+    else
+      # We get all DIRECT subclasses of ApplicationRecord, rather than all descendents
+      # since the table containing source strings will be a direct subclass of
+      # ActiveRecord. We don't need to process derived models.
+      klasses = ApplicationRecord.subclasses.select { |m| m.included_modules.include?(Mobility::ActiveRecord) }
```

Comment:
cool, didn't know you can get descendent classes like so

Reply:
neither did i. seems like it's new with more recent versions of Rails.